ARCs/arc-0014.md

```diff
@@ -0,0 +1,67 @@
+---
+ams: 14
+title: Algorand standard for Authentication
+status: Draft
+---
+
+# Algorand Standard for authentication
+
+## Summary
+
+This document introduces the standard for authentication using the algorand accounts.
+
+## Abstract
+
+The goal of this standard is to define authorization process and authentication procedures for communication between web application and backend services or between two backend services. 
+
+## Specification
+
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Base64 is the standard described in [RFC-4648](https://www.ietf.org/rfc/rfc4648.txt).
+
+## Request for comments
+
+Please comment or suggest pull request for this document here: [https://github.com/scholtz/AMS/issues](https://github.com/scholtz/AMS/issues)
+
+### Authentication
+
+Authentication is process in which person generates the authorization token for communication with backend service.
```

Comment:
I find the terminology confusing. In authorisation code flow authentication typically yields an Identity Token and a code, which can ultimately be exchanged for Access Tokens. In my mind authentication and authorisation are unrelated.